### PR TITLE
CNF-10662: ci:nto: add TechPreviewNoUpgrade job

### PR DESCRIPTION
**Pull request by Tal-or**

### CODE REVIEW

**Comment by Tal-or** [comment]
depend on: https://github.com/openshift/cluster-node-tuning-operator/pull/883

**Comment by Tal-or** [comment]
/pj-rehearse pull-ci-openshift-cluster-node-tuning-operator-master-e2e-aws-ovn-techpreview

**Comment by Tal-or** [comment]
/pj-rehearse pull-ci-openshift-cluster-node-tuning-operator-master-e2e-aws-ovn-techpreview

File: ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-master.yaml [hunk 1]
```diff
@@ -83,6 +83,22 @@ tests:
...
+- as: e2e-aws-ovn-techpreview
+  steps:
+    cluster_profile: aws
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+    test:
+    - as: e2e-pao-mixed-cpus
+      cli: latest
+      commands: CLUSTER=mcp-only make cluster-label-worker-cnf cluster-deploy-pao
+        pao-functests-mixedcpus
```
Comment (swatisehgal) [inline ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-master.yaml:95]: Why in a new line? Commands for all the other jobs are in a single line, should we do the same here?

**Review by swatisehgal — APPROVED** [review]
/lgtm

Left a minor comment about execution command split into multiple lines.

File: ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-master.yaml (hunk 1 above)
Comment (Tal-or) [reply ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-master.yaml:95]: The call to `make ci-operator-config`  added the new automatically.
We probably crossed the configured threshold here.

**Comment by jmencak** [comment]
Thank you for the PR.
/approve

File: ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-master.yaml (hunk 1 above)
Comment (swatisehgal) [reply ci-operator/config/openshift/cluster-node-tuning-operator/openshift-cluster-node-tuning-operator-master.yaml:95]: Ack, Thanks for clarifying.

**Pull request merged**